manifests/init.pp

```diff
@@ -247,6 +247,14 @@
   $apm_env = '',
 ) inherits datadog_agent::params {
 
+  # Allow ports to be passed as integers or strings.
+  $_statsd_forward_port = "$statsd_forward_port"
```

Comment:
This will probably have to look like this to pass the linter:

```
  # lint:ignore:only_variable_string
  $_statsd_forward_port = "${statsd_forward_port}"
  $_proxy_port = "${proxy_port}"
  $_graphite_listen_port = "${graphite_listen_port}"
  $_listen_port = "${listen_port}"
  $_pup_port = "${pup_port}"
  $_syslog_port = "${syslog_port}"
  # lint:endignore
```